Fix compiling error:
COpenGLDriver.cpp: In member function `void irr::video::COpenGLDriver::loadExtensions()':
COpenGLDriver.cpp:357: error: `glXGetProcAddress' undeclared (first use this function)
COpenGLDriver.cpp:357: error: (Each undeclared identifier is reported only once for each function it appears in.)
make: *** [COpenGLDriver.o] Error 1

Patch by Chris Statzer.

http://bugs.gentoo.org/87015

--- COpenGLDriver.cpp
+++ COpenGLDriver.cpp
@@ -2,6 +2,7 @@
 
 #include "IrrCompileConfig.h"
 #ifdef _IRR_COMPILE_WITH_OPENGL_
+#define glXGetProcAddress glXGetProcAddressARB
 
 #include "COpenGLTexture.h"
 #include "COpenGLMaterialRenderer.h"
